packages/react/src/themes/teams-dark/components/Input/inputVariables.ts

```diff
@@ -0,0 +1,5 @@
+import { InputVariables } from '../../../base/components/Input/inputVariables'
+
+export default (siteVars): Partial<InputVariables> => ({
+  backgroundColor: siteVars.colors.grey[650],
```

Comment:
@codepretty is this valid change?

Reply:
No, we don't want to do this. There are valid cases when we want the input to have that background color. I added `inverted` prop to Input in https://github.com/stardust-ui/react/pull/1684. This is what should be used when the input should swap background colors. I updated the [issue](https://github.com/stardust-ui/react/issues/1870) as well.